ci/gpu/build.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# Copyright (c) 2018, NVIDIA CORPORATION.
```

Comment:
@younseojava 
* `Modifications Copyright (c) 2024 Advanced Micro Devices, Inc.` missing.
* AMD License missing. Ideally include the original `dask_cuda` license below the NVIDIA copyright note.